x/gov/keeper/proposal_test.go

```diff
@@ -232,6 +276,7 @@ func (suite *KeeperTestSuite) TestCancelProposal() {
 
 	for _, tc := range testCases {
 		suite.Run(tc.name, func() {
+			tc.malleate()
 			if tc.proposalID == proposal2ID {
```

Comment:
nit: since you introduced `malleate`, can you migrate this condition to make use of it?